learn/advanced/filtering_and_faceted_search.md

```diff
@@ -195,49 +321,29 @@ Suppose that you have a dataset containing several movies in the following forma
 ]
 ```
 
-If you want to enable filtering using `director`, `release_date`, `genres`, and `rating.users`, you must add these attributes to the [`filterableAttributes` index setting](//reference/api/settings.md#filterable-attributes).
-
-You can then restrict a search so it only returns movies released after 18 March 1995 with the following filter containing a single condition:
+After adding `director`, `release_date`, and `genres` to the [`filterableAttributes` index setting](//reference/api/settings.md#filterable-attributes), you can use them for filtering.
 
-```SQL
-release_date > 795484800
-```
-
-You can use this filter when searching for recent `Avengers` movies:
+The following code sample returns `Avengers` movies released after 18 March 1995:
 
 <CodeSamples id="filtering_guide_1" />
 
-You can also combine multiple conditions. For instance, you can limit your search so it only includes recent movies directed by either `Tim Burton` or `Christopher Nolan`:
-
-```SQL
-release_date > 795484800 AND (director = "Tim Burton" OR director = "Christopher Nolan")
-```
-
-Here, the parentheses are mandatory: without them, the filter would return movies directed by `Tim Burton` and released after 1995 or any film directed by `Christopher Nolan`, without constraints on its release date. This happens because `AND` takes precedence over `OR`.
-
-You can use this filter when searching for `Batman` movies:
+You can also combine multiple conditions. For instance, you can limit your search so it only includes `Batman` movies directed by either `Tim Burton` or `Christopher Nolan`:
```

Comment:
Minor, but I remember us discussing a long time ago that we perhaps should reserve "instance" for when we are discussing Meilisearch instances.

```suggestion
You can also combine multiple conditions. For example, you can limit your search so it only includes `Batman` movies directed by either `Tim Burton` or `Christopher Nolan`:
```